src/entropies_estimators/nearest_neighbors/GaoNaive.jl

```diff
@@ -0,0 +1,86 @@
+using StateSpaceSets: AbstractDataset
+using Neighborhood: Euclidean, KDTree, NeighborNumber, Theiler
+using Neighborhood: bulksearch
+using SpecialFunctions: digamma
+
+export GaoNaive, GaoNaiveCorrected
+
+"""
+    GaoNaive <: DifferentialEntropyEstimator
+    GaoNaive(; k = 1, w = 0)
```

Comment:
What's up with this file...?

Reply:
Good question. I branched this off while still not having merged `GaoNaive` and `GaoNaiveCorrected` and forgot to remove it. Will fix asap.

Reply:
There were also some extra test files here that weren't removed before. I fixed that too.